### PR TITLE
Show better looking help when a wrong flag is passed

### PR DESCRIPTION
**Pull request by vadmeste**
To test, type `mc admin user add -zz`

This change just implements onUsageError() function and adds it to cli.Command to various available commands.

Unfortunately there is no way to support `mc admin user -zz` or change its behavior.

### CODE REVIEW

File: cmd/mb-main.go [hunk 1]
```diff
@@ -153,6 +155,8 @@ func mainMakeBucket(cli *cli.Context) error {
 			continue
 		}
 
+		fmt.Println(clnt.GetURL())
```
Comment (poornas) [inline cmd/mb-main.go:158]: needs to be removed

**Review by poornas — APPROVED** [review]
LGTM. Println needs to be removed from mb command

**Review by ebozduman — APPROVED** [review]
LGTM

File: cmd/mb-main.go (hunk 1 above)
Comment (harshavardhana) [inline cmd/mb-main.go:158]:
```suggestion
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

File: cmd/mb-main.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ package cmd
 
 import (
 	"context"
+	"fmt"
```
Comment (harshavardhana) [inline cmd/mb-main.go:21]:
```suggestion
```

**Pull request merged**